.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      env:
+        KIND_VERSION: v0.12.0
+        KIND_OS: kind-linux-amd64
+      run: |
+        curl -sLo kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/${KIND_OS}"
+        chmod +x kind
+        sudo mv kind /bin/
+
+    - name: start_kind_cluster
+      run: |
+        # /bin/kind create cluster
+        make kind-cluster
```

Comment:
I think we may also need to export the kubeconfig per https://github.com/operator-framework/operator-lifecycle-manager/blob/3002cf79ce867e32d30d2190df642eaa7f449294/.github/workflows/run-kind-local.yml#L23

Reply:
Done